src/components/services/controllers.tsx

```diff
@@ -72,3 +76,115 @@ export async function viewService(
     ),
   };
 }
+
+export async function listServiceLogs(ctx: IContext, params: IParameters): Promise<IResponse> {
+  const cf = new CloudFoundryClient({
+    accessToken: ctx.token.accessToken,
+    apiEndpoint: ctx.app.cloudFoundryAPI,
+    logger: ctx.app.logger,
+  });
+
+  const [userProvidedServices, space, organization] = await Promise.all([
+    cf.userServices(params.spaceGUID),
+    cf.space(params.spaceGUID),
+    cf.organization(params.organizationGUID),
+  ]);
+
+  const isUserProvidedService = userProvidedServices.some(
+    s => s.metadata.guid === params.serviceGUID,
+  );
+
+  if (isUserProvidedService) {
+    throw new UserFriendlyError(UNSUPPORTED_SERVICE_LOGS_REQUEST);
+  }
+
+  const serviceInstance = await cf.serviceInstance(params.serviceGUID);
+  const service = await cf.service(serviceInstance.entity.service_guid);
+
+  if (!['postgres', 'mysql'].includes(service.entity.label)) {
+    throw new UserFriendlyError(UNSUPPORTED_SERVICE_LOGS_REQUEST);
+  }
+
+  const rds = new RDS();
+  const fileList = await rds.describeDBLogFiles({
```

Comment:
I'm assuming AWS returns these in the correct order, so we don't have to paginate

Reply:
It would appear that AWS returns all the files (162 at once) in a asc order. Hence the following `.reverse()`.